javascriptv3/example_code/lambda/README.md

```diff
@@ -1,12 +1,15 @@
 # TypeScript environment for Amazon Lambda examples
-Environment for AWS SDK for JavaScript (V3) Amazon Lambda tutorial. For more information, see the [AWS documentation for this tutorial](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/using-lambda-functions.html).
+Environment for AWS SDK for JavaScript (v3) Amazon Lambda tutorial. 
```

Comment:
Amazon Lambda -> AWS Lambda, here and throughout.

Reply:
Done